src/main/java/com/ust/sourcecourse/configuration/service/ProjectGroupService.java

```diff
@@ -112,4 +118,69 @@ public boolean deleteProjectGroup(Long uid) {
 			return false;
 		}
 	}
+
+	/**
+	 * 
+	 * @param Add tags
+	 * @return
+	 */
+
+	public ProjectGroup addTagToProjectGroup(Long uid, List<String> tags) {
+		ProjectGroup projectGroup = projectGroupRepository.findById(uid)
+				.orElseThrow(() -> new ResourceNotFoundException("ProjectGroup", "id", uid));
+		List<String> tag = projectGroup.getTags();
+		if (tag == null) {
+			tag = new ArrayList<>();
+		}
+		tag.addAll(tags);
```

Comment:
We need to handle duplicate. If same tag is in the request, we should not add again. 
A set can be used